### ISSUE TITLE
[Feature Request] Support to display the bare post content.

### ISSUE THREAD

**Issue by dohooo** [Crossbell-Box/xLog#337]
We need to display the post content on the post page in the xlog-mobile app. Therefore, I suggest that the xLog project should provide a new page that only includes the post content and does not contain any authentication content or actions.

**Comment by Songkeys** [comment]
Why not render it directly on the RN end?

**Comment by dohooo** [comment]
Here are my opinions:
- Currently, there are no good React Native projects that support Markdown rendering.
- Using an embedded WebView makes it easier for users to customize styles.
- At present, displaying the details page through WebView is simpler, and I think this issue has a low priority and there won't be much difference in the presentation compared to native rendering.
- For new features added to the web, such as replacing <APlayer/>, the App does not need to release a new version. Otherwise, the app would experience rendering errors due to the lack of compatible components.

In summary, I believe this is a suitable solution in terms of maintainability and development efficiency, without sacrificing too much user experience. 🤔
